webpack/scenes/Subscriptions/Manifest/index.js

```diff
@@ -11,15 +12,20 @@ import ManifestModal from './ManageManifestModal';
 
 import './Manifest.scss';
 
+export { MANAGE_MANIFEST_MODAL_ID } from './ManifestConstants';
```

Comment:
why? all the components import it from `'./ManifestConstants'` anyway

Reply:
@MariaAga true, removing